api/resolvers/item.js

```diff
@@ -802,7 +809,6 @@ export default {
         return await updateItem(parent, { id, ...item }, { me, models })
```

Comment:
To test edits that cost something, I updated the image fees function such that an image always costs 100 sats (like for anons). I noticed that `lnd` was undefined inside `createDbInvoice` due to this:

```suggestion
        return await updateItem(parent, { id, ...item }, { me, models, lnd })
```

Same for line 800, here is a full patch since I can't comment on line 800:

```diff
diff --git a/api/resolvers/item.js b/api/resolvers/item.js
index 95e76820..f83382d0 100644
--- a/api/resolvers/item.js
+++ b/api/resolvers/item.js
@@ -797,7 +797,7 @@ export default {
       item.maxBid ??= 0

       if (id) {
-        return await updateItem(parent, { id, ...item }, { me, models })
+        return await updateItem(parent, { id, ...item }, { me, models, lnd })
       } else {
         return await createItem(parent, item, { me, models, lnd, hash, hmac })
       }
@@ -806,7 +806,7 @@ export default {
       await ssValidate(commentSchema, item)

       if (id) {
-        return await updateItem(parent, { id, ...item }, { me, models })
+        return await updateItem(parent, { id, ...item }, { me, models, lnd })
       } else {
         item = await createItem(parent, item, { me, models, lnd, hash, hmac })
         return item
```

Reply:
these also need hash and hmac. Nice catch